src/components/MasterItems.js

```diff
@@ -8,36 +9,28 @@ class MasterItems extends Component {
   constructor(props) {
     super(props);
     this.selectItem = this.selectItem.bind(this);
-    this.addComment = this.addComment.bind(this);
     this.handleAddEntry = this.handleAddEntry.bind(this);
     this.removeEntry = this.removeEntry.bind(this);
     this.handleOnClick = this.handleOnClick.bind(this);
     this.renderSelectedEntries = this.renderSelectedEntries.bind(this);
     this.renderNoEntries = this.renderNoEntries.bind(this);
+    this.addComment = this.addComment.bind(this);
     this.state = {
-      comment: '',
       entries: []
     };
   }
 
   handleAddEntry() {
-    const selectedItem = _.extend({}, this.state.selectedItem || this.props.masterItems[0]);
-    selectedItem.comment = this.state.comment;
+    const selectedItem = R.merge({}, this.state.selectedItem || this.props.masterItems[0]);
     const entries = this.state.entries.concat(selectedItem);
 
-    this.setState({ entries, comment: '' });
+    this.setState({ entries });
   }
 
   selectItem(selectedItem) {
     this.setState({ selectedItem });
   }
 
-  addComment(e) {
-    const comment = e.target.value;
-
-    this.setState({ comment });
-  }
-
   removeEntry(entry) {
     const updatedEntries = _.without(this.state.entries, entry);
```

Comment:
yeah ramda's without removes all instances of the same object, while underscore's removes just one